### PR TITLE
Documentation/41133 improve documentation for configuration variables to include allowed values

### PR DESCRIPTION
**Pull request by ghost**
[#41133] Improve documentation for configuration variables to include allowed values

https://community.openproject.org/work_packages/41133

### CODE REVIEW

File: docs/installation-and-operations/configuration/environment/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ sidebar_navigation:
 
 When using environment variables, you can set the options by setting environment variables with the name of the options below in uppercase. So for example, to configure email delivery via an SMTP server, you can set the following environment variables:
 
-```bash
+```ruby
```
Comment (machisuji) [inline docs/installation-and-operations/configuration/environment/README.md]: This isn't ruby! We should leave this at bash.

File: docs/installation-and-operations/configuration/environment/README.md [hunk 2]
```diff
@@ -61,82 +67,253 @@ Please note: The Configuration is a HashWithIndifferentAccess and thus it should
 
 # Supported environment variables
 
-Below is the full list of supported environment variables that can be used to override the default configuration of your OpenProject installation:
+A list of all configuration variables that can be overridden by defining them in the environment from bash could be generated by executing the following rake task:
+
+For packaged installations:
+
+```bash
+sudo openproject run bundle exec rake 'setting:available_envs'
```
Comment (machisuji) [inline docs/installation-and-operations/configuration/environment/README.md]: You don't need the quotes around `setting:available_envs`.

File: docs/installation-and-operations/configuration/environment/README.md [hunk 3]
```diff
@@ -61,82 +67,253 @@ Please note: The Configuration is a HashWithIndifferentAccess and thus it should
...
+docker exec -it $(docker ps | grep web | cut -d' ' -f 1) bash -c "RAILS_ENV=production bundle exec rake 'setting:available_envs'"
 ```
-OPENPROJECT_EDITION (default="standard")                                                                                                         
-OPENPROJECT_ATTACHMENTS__STORAGE (default="file")                                                                                                 
+
+The default value is also visible for each variable in parenthesis.
+
+```ruby
+#OpenProject Version 12.2.1
+
```
Comment (machisuji) [inline docs/installation-and-operations/configuration/environment/README.md:85]: I would suggest adding a comment here that the following are the correct values at the time of writing and that you should use the command shown above to make sure to get the up-to-date list.

**Review by machisuji — CHANGES_REQUESTED** [review]
Thanks @adam-op. Looking good overall. I just have some minor nit-picks.

File: docs/installation-and-operations/configuration/environment/README.md (hunk 2 above)
Comment (ghost) [reply docs/installation-and-operations/configuration/environment/README.md]: was corrected

File: docs/installation-and-operations/configuration/environment/README.md (hunk 1 above)
Comment (ghost) [reply docs/installation-and-operations/configuration/environment/README.md]: was corrected

File: docs/installation-and-operations/configuration/environment/README.md (hunk 3 above)
Comment (ghost) [reply docs/installation-and-operations/configuration/environment/README.md:85]: was added as suggested
@machisuji please check again

**Review by machisuji — APPROVED** [review]
👍

**Pull request merged**